mcodingbot/bot.py

```diff
@@ -29,6 +29,33 @@ def __init__(self) -> None:
         self._session: aiohttp.ClientSession | None = None
         self._db: Database | None = None
 
+        if not CONFIG.mcoding_server:
+            _LOG.warning(
+                "Server stats and donor roles will not be updated because"
+                " the mcoding server id is not provided. Is this intended?"
+            )
+        else:
+            if not CONFIG.sub_count_channel:
+                _LOG.warning(
+                    "Sub count channel id for mcoding server required to post"
+                    " stats."
+                )
+            if not CONFIG.view_count_channel:
+                _LOG.warning(
+                    "View count channel id for mcoding server required to post"
+                    " stats."
+                )
+            if not CONFIG.member_count_channel:
+                _LOG.warning(
+                    "Member count channel id for mcoding server required to"
+                    " post stats."
+                )
+            if not CONFIG.donor_role:
+                _LOG.warning(
+                    "Doner role id for mcoding server expected to update"
+                    " donor roles."
+                )
```

Comment:
Can't you DRY this? Also "Doner role" 🇹🇷

Reply:
Idk a way to make it DRYer and keep it as readable

Reply:
Now that I read these I feel like it should match the variable name exactly ("view_count_channel_id" not "View count channel id")

Reply:
I tried drying this and I made the changes circuit requested